app/modules/encounters/schemas.py

```diff
@@ -21,8 +21,8 @@ class BaseEncounterSchema(ModelSchema):
     owner = base_fields.Nested('PublicUserSchema', many=False)
     hasView = base_fields.Function(Encounter.current_user_has_view_permission)
     hasEdit = base_fields.Function(Encounter.current_user_has_edit_permission)
-    time = base_fields.Function(Encounter.get_time_isoformat_in_timezone)
-    timeSpecificity = base_fields.Function(Encounter.get_time_specificity)
+    time = base_fields.Function(lambda enc: enc.get_time_isoformat_in_timezone())
```

Comment:
Let's migrate these to properties, a la: [model](https://github.com/WildMeOrg/houston/blob/b6a496ec9a504fae5a4b5975de97289c3eb4a2eb/app/modules/assets/models.py#L199) and [schema](https://github.com/WildMeOrg/houston/blob/b6a496ec9a504fae5a4b5975de97289c3eb4a2eb/app/modules/assets/schemas.py#L65)

Reply:
Overall, I generally dislike the `base_fields.Function` usage because it leads to more complex usage in corner cases.  The downside is we have to wrap things with lambda functions when we functionally are treating this as a property from the standpoint of the schema.  I'd argue if we are treating a function like a property for schema serialization, we might as well make it a formal property on the object and make it something we can use more universally in the Python code.  If it is useful to be a property for serializing, it's likely useful to be a property in general on the object.

Reply:
Yeah, I would consider using `@property` as well except we already have a database field called `time`.  Or do you think we should change the keys from `time` to something else?